sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingClientBuilder.java

```diff
@@ -242,6 +244,18 @@ public FormTrainingClientBuilder credential(AzureKeyCredential apiKeyCredential)
         return this;
     }
 
```

Comment:
Add the credential in builder docs as well. Same as [this](https://github.com/Azure/azure-sdk-for-java/pull/11156/files#diff-fb4c077c9efd724485fb17f300221edcR48)

Reply:
updated